.github/workflows/file-system-plugin-py-cd.yaml

```diff
@@ -0,0 +1,59 @@
+name: file-system-plugin-py-cd
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - main
+    paths:
+      - "implementations/py/**"
+
+jobs:
+  cd:
+    name: file-system-plugin-py-cd
+    if: |
+      github.event.pull_request.merged &&
+      startsWith(github.event.pull_request.title, '[CD]') == true &&
+      github.event.pull_request.user.login != 'github-actions'
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Read .nvmrc
+        run: echo ::set-output name=NVMRC::$(cat .nvmrc)
+        id: nvm
+        working-directory: ./implementations/py
```

Comment:
should we have the `.nvmrc` in the python implementation? i think we should use the one from the root - wdyt?

Reply:
I think it's alright!